tests/unit/src/test/scala/tests/JarTopLevelsSuite.scala

```diff
@@ -0,0 +1,227 @@
+package tests
+
+object JarTopLevelsSuite extends BaseSlowSuite("JarTopLevelsSuite") {
```

Comment:
Instead of slow tests, can we have unit tests for `JarTopLevels`? The tests below below duplicate a lot from `DefinitionSlowSuite`, since this feature is enabled by default it is tested anyways in all existing slow suites.

Reply:
Ok, I'll move it and convert into unit tests. You're right it was a lazy way of doing the tests.

Reply:
Unit tests added, now more slow tests.